Add Indie

mezod opened the pull request:
<!-- Congrats on creating an Awesome list! 🎉 -->


<!-- Please fill in the below placeholders -->

**[Insert URL to the list here]**
https://github.com/mezod/awesome-indie
**[Explain what this list is about and why it should be included here]**
It's a list of resources for developers who want to monetize their sideprojects or bootstrap companies

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Has the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- Each link description starts with an uppercase character and ends with a period.<br>Example: `- [AVA](…) - JavaScript test runner.`
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

sindresorhus commented on readme.md:
```diff
@@ -420,6 +420,7 @@
 - [Open Companies](https://github.com/opencompany/awesome-open-company)
 - [Places to Post Your Startup](https://github.com/mmccaff/PlacesToPostYourStartup)
 - [OKR Methodology](https://github.com/domenicosolazzo/awesome-okr) - Goal setting & communication best practices.
+- [Indie](https://github.com/mezod/awesome-indie) - Resources to help independent developers make money.
```
"Resources" is implied with it being an awesome list. Maybe:

> Making money as an independent developer

sindresorhus commented:
Some feedback:

- The title should be `awesome-indie` => `Awesome Indie` and the Awesome should be on the right side.
- `Table of Contents` => `Contents`
- The `contributing.md` should be adapted to your use-case, not just copy-pasted.
- `Resources to help independent developers make money.` => `> Resources to help independent developers make money.`

---

> Each link description starts with an uppercase character and ends with a period.

:arrow_up:

mezod commented:
sorry, I updated the readme but I'm not sure how to resend the pull request, does it already take the changes?

1. I updated the title.
2. Updated to Contents
3. The contributing is already adapted. I took the list from someone else (maybe, you?) because it makes sense and it's the formatting I used!
4. I updated the description so that it doesn't contain the word "Resources". I am not sure if this is what you meant here.
5. Each link description starts with an uppercase character and ends with a period.

sindresorhus commented:
> The contributing is already adapted. I took the list from someone else (maybe, you?) because it makes sense and it's the formatting I used!

I noticed because it says `Use the following format: [List Name](link)` Notice "List Name".

> I updated the description so that it doesn't contain the word "Resources". I am not sure if this is what you meant here.

Yes, I must have copied the wrong thing.
